src/components/AssetInfo/TokenSideBar.test.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { mount } from "enzyme";
+import TokenSideBar from "./TokenSideBar";
+
+describe("tokenSideBar", () => {
+  it("should have a Manage Asset heading", () => {
+    const wrapper = mount(
+      <TokenSideBar
+        isSideBarExpand={true}
+        holderAddress=""
+        beneficiaryAddress=""
+        approvedBeneficiaryAddress=""
+        handler={() => {}}
+      />
+    );
+    expect(wrapper.find("h2").text()).toStrictEqual("Manage Asset");
+  });
+
+  // holder === bene
+  it("should show holder and beneficiary role, holder view, only 3 correct fields", () => {
+    const wrapper = mount(
+      <TokenSideBar
+        isSideBarExpand={true}
+        holderAddress="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
+        beneficiaryAddress="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
+        approvedBeneficiaryAddress="0xC"
+        handler={() => {}}
+      />
+    );
+
+    expect(wrapper.find("TokenSideBarRole h4").text()).toStrictEqual("Holder and Beneficiary");
+    expect(wrapper.find("TokenSideBarHolder")).toHaveLength(1);
+    expect(wrapper.find("TokenSideBarHolder section")).toHaveLength(3);
+
+    expect(wrapper.find("#sec-transferownership h4").text()).toStrictEqual("Transfer Holdership");
+    expect(wrapper.find("#sec-transferownership button").text()).toStrictEqual("Transfer");
+
+    expect(wrapper.find("#sec-changebeneficiary h4").text()).toStrictEqual("Change Beneficiary");
+    expect(wrapper.find("#sec-changebeneficiary button").text()).toStrictEqual("Submit");
+
+    expect(wrapper.find("#sec-surrenderdocument h4").text()).toStrictEqual("Surrender Document");
+    expect(wrapper.find("#sec-surrenderdocument button").text()).toStrictEqual("Surrender");
+  });
+
+  // admin address === holder
+  it("should show holder role, holder view, only 1 correct field", () => {
+    const wrapper = mount(
+      <TokenSideBar
+        isSideBarExpand={true}
+        holderAddress="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
+        beneficiaryAddress=""
+        approvedBeneficiaryAddress=""
+        handler={() => {}}
+      />
+    );
+    expect(wrapper.find("TokenSideBarRole h4").text()).toStrictEqual("Holder");
+    expect(wrapper.find("TokenSideBarHolder")).toHaveLength(1);
+    expect(wrapper.find("TokenSideBarHolder section")).toHaveLength(1);
+
+    expect(wrapper.find("#sec-transferownership h4").text()).toStrictEqual("Transfer Holdership");
+    expect(wrapper.find("#sec-transferownership button").text()).toStrictEqual("Transfer");
```

Comment:
TODO: make issue to review terminology with UXD

Reply:
Boon made an issue for this.